firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedFid.java

```diff
@@ -62,30 +63,38 @@ public FiidCache(@NonNull FirebaseApp firebaseApp) {
   }
 
   @Nullable
-  public synchronized FiidCacheEntryValue readCacheEntryValue() {
-    String iid = prefs.getString(getSharedPreferencesKey(FIREBASE_INSTALLATION_ID_KEY), null);
-    int status = prefs.getInt(getSharedPreferencesKey(CACHE_STATUS_KEY), -1);
+  public synchronized PersistedFidEntry readPersistedFidEntryValue() {
+    String fid = prefs.getString(getSharedPreferencesKey(FIREBASE_INSTALLATION_ID_KEY), null);
+    int status = prefs.getInt(getSharedPreferencesKey(PERSISTED_STATUS_KEY), -1);
     String authToken = prefs.getString(getSharedPreferencesKey(AUTH_TOKEN_KEY), null);
     String refreshToken = prefs.getString(getSharedPreferencesKey(REFRESH_TOKEN_KEY), null);
     long tokenCreationTime =
         prefs.getLong(getSharedPreferencesKey(TOKEN_CREATION_TIME_IN_SECONDS_KEY), 0);
     long expiresIn = prefs.getLong(getSharedPreferencesKey(EXPIRES_IN_SECONDS_KEY), 0);
 
-    if (iid == null || status == -1) {
+    if (fid == null || status == -1) {
```

Comment:
Why is this not a constant?
What does -1 mean?

Reply:
we pass -1 as a default in the call to prefs.getInt